agent/config/config_windows.go

```diff
@@ -57,6 +57,8 @@ func DefaultConfig() Config {
 	programData := utils.DefaultIfBlank(os.Getenv("ProgramData"), `C:\ProgramData`)
 	ecsRoot := filepath.Join(programData, "Amazon", "ECS")
 	dataDir := filepath.Join(ecsRoot, "data")
+	DefaultPauseContainerImageName = "amazonaws.com/ecs/pause-windows"
```

Comment:
Shouldn't this be a const or a var? 
Also should there be a mechanism for customers to override the `defaultPauseContainer`?

Reply:
As discussed, I have created environment variables for the pause image name and tag. If the same are not available then the default values are considered.
I have moved the constants to const_windows.go

Reply:
It's a var because the linux version gets populated by the Makefile.

I'm curious why was this same pattern not followed? Why hardcoded for windows and populated by build flag for linux?

Reply:
As discussed with the agent team, we have added the Windows pause image name and tag in the Makefile and corresponding changes have been made in build file to invoke Windows agent build using the loader flags. The default values for both name and tag will be injected using the loader flag.

Additionally, a Powershell script has been created which can build standalone agent on Windows.